WalkingChallenge/Notifications/Notifications.swift

```diff
@@ -56,17 +56,31 @@ class Notifications: UIViewController {
     }
 
     self.tableView.dataSource = self
+    self.tableView.delegate = self
+    self.tableView.register(NotificationTableViewCell.self, forCellReuseIdentifier: "cell")
     self.tableView.tableFooterView = UIView()
   }
 }
 
 extension Notifications: UITableViewDataSource {
   func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-    return 1
+    //TODO: update with data
+    return 3
   }
 
   func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-    let cell = UITableViewCell()
+    guard let cell = tableView.dequeueReusableCell(withIdentifier: "cell",
```

Comment:
Can you make "cell" a static member of the cell type.  Please use a better string (e.g. "NotificationCell").

Reply:
got it so.. 
`static let CellIdentifier = "NotificationCell"` work?

Reply:
Yeah, that should be fine.  Please type the variable as well (`String`).